dhcpv6/option_ntp_server.go

```diff
@@ -0,0 +1,58 @@
+package dhcpv6
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/u-root/uio/uio"
+)
+
+// NTPSuboptionSrvAddr is the value of NTP_SUBOPTION_SRV_ADDR according to RFC 5908.
+const NTPSuboptionSrvAddr byte = 1
+
+// OptNTPServer returns an NTP Server option as defined by RFC 5908.
+func OptNTPServer(ntpServers ...net.IP) Option {
+	return &optNTPServer{NTPServers: ntpServers}
+}
+
+type optNTPServer struct {
+	NTPServers []net.IP
```

Comment:
This is not enough information for the RFC defined option.

I think this should be a an option list like in: https://github.com/insomniacslk/dhcp/blob/d4b282fe8d4cf3f8023e01e0f17cb74c510ad228/dhcpv4/option_relay_agent_information.go#L10

Reply:
I messed up reading the RFC. Fixing